packages/block-library/src/classic/block.native.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "core/freeform",
+	"category": "formatting",
+	"attributes": {
+		"content": {
+			"type": "string",
+			"source": "html"
+		}
+	},
+	"supports": {
+		"className": false,
+		"customClassName": false,
+		"inserter": false,
```

Comment:
I guess that’s the only difference with web definition. It might be easier to maintain it if it was applied with WordPress hook when the registration happens. It’s more code but you would ensure that the block definition always stays in sync.